content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: How Kubernetes contributors are building a better communication process
+date: 2020-04-21
+slug: contributor-communication
+---
+
+**Authors:** Paris Pittman
+
+> “Perhaps we just need to use a different word. We may need to use community development or project advocacy as a word in the open source realm as opposed to marketing, and perhaps then people will realize that they need to do it.” 
+> ~ [*Nithya Ruff*](https://todogroup.org/www.linkedin.com/in/nithyaruff/) (from [*TODO Group*](https://todogroup.org/guides/marketing-open-source-projects/))
+
+A common way to participate in the Kubernetes contributor community is
+to be everywhere. 
+
+We have an active Slack, many mailing lists, Twitter account(s), and
+dozens of community-driven podcasts and newsletters that highlight all
+end-user, contributor, and ecosystem topics. And guess what else - an
+entire repository of amazing documentation that I bet there is one gem
+you didn’t know we had! All of this information is incredibly valuable,
+but it can be too much. 
+
+Keeping up with thousands of contributors can be a challenge for anyone.
+but this task of consuming information straight from the firehose is
+particularly challenging for new community members. It’s no secret that
+the project is vast for contributors and users alike. 
+
+To paint a picture with numbers: 
+
+- 43,000 contributors
+- 6,579 members in #kubernetes-dev slack channel
+- 52 mailing lists (kubernetes-dev@ has thousands of members; sig-networking@ has 1000 alone)
+- 40 [community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups)
+- 30 [upstream meetings](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America%2FLos_Angeles) *this* week alone
+
+The scale is growing, and with that comes the need for attention.
```

Comment:
How about something like "The scale of the project is growing, and with that comes the need to ensure contributors are able to get information about important tasks at hand.", or "important information is lifted up front-and-center for contributors" ?

Reply:
thanks - went with your suggestion and combo'ed those